src/OpenTelemetry.Api/README.md

```diff
@@ -46,10 +48,17 @@ conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/
 API](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md)
 allows users to generate
 [Spans](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#span),
-which represent a single operation within a trace. Spans can be nested to form
-a trace tree. Each trace contains a root span, which typically describes the
+which represent a single operation within a trace. Spans can be nested to form a
+trace tree. Each trace contains a root span, which typically describes the
 entire operation and, optionally one or more sub-spans for its sub-operations.
```

Comment:
Not introduced by this PR, I wonder if we want to call it "child spans" instead of "sub spans" (and "child operations")?

Reply:
Yea agree child spans sounds better.